cases/transactions.test.js

```diff
@@ -166,6 +166,14 @@ describe("Transactions", () => {
       isDeposit: false,
     });
 
+    transactionJson = await getTransactionBy({
```

Comment:
I don't think you need this call, `transactionJson.transaction` already has `started_at` when its returned from `createTransaction`.

Reply:
That's how I implemented it at first, but this is what the createTrasaction returns:

` {
      transaction: {
        id: '17b6b4d1-b77f-4f78-b8fa-5590e6469857',
        kind: 'deposit',
        status: 'incomplete',
        status_eta: 3600,
        amount_in: null,
        amount_out: null,
        amount_fee: null,
        started_at: '2020-04-08T17:21:54.831026Z',
        completed_at: null,
        stellar_transaction_id: null,
        external_transaction_id: null,
        external_extra: null,
        external_extra_text: null,
        deposit_memo: null,
        deposit_memo_type: 'text',
        more_info_url: 'https://testanchor.stellar.org/transaction/more_info?id=17b6b4d1-b77f-4f78-b8fa-5590e6469857',
        refunded: false,
        message: 'incomplete',
        to: 'GDLJINBHLJA2XKG3HLKMZOMR4AGJ4TWSECIJIU4CCEBYBYXVM7AE2OB7',
        from: null
      }
    }`

Reply:
I see `started_at`?

Reply:
Inside the transaction object, yes

Reply:
Oh, sorry, I misunderstood you

Reply:
The createTransaction returns:

`    {
      type: 'interactive_customer_info_needed',
      url: 'https://testanchor.stellar.org/transactions/deposit/webapp?asset_code=SRT&transaction_id=f58f3899-ad39-4c63-b44a-3a78565f21f1&token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJodHRwczovL3Rlc3RhbmNob3Iuc3RlbGxhci5vcmcvdHJhbnNhY3Rpb25zL2RlcG9zaXQvaW50ZXJhY3RpdmUiLCJpYXQiOjE1ODYzNjU2NjMuODY5NTc3MiwiZXhwIjoxNTg2MzY1NjkzLjg2OTU3NzIsInN1YiI6IkdESFJDQ0JVNzZKNjdVRk1RT1laU0ZZQ0lRVVBQQTdEQVNIUldISlRHNzRXTzM2S0hCNDU3QjRDIiwianRpIjoiZjU4ZjM4OTktYWQzOS00YzYzLWI0NGEtM2E3ODU2NWYyMWYxIn0.z6AD4FA2RJErla2lDmuCKQna2AMPPBOXSs4Z8mSYBEI',
      id: 'f58f3899-ad39-4c63-b44a-3a78565f21f1'
    }`

Reply:
Ohh right.. ok nevermind, looks good then!